page/create_join_link.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  page: page
+}
+
+function page (req, res) {
+  // debug function to get links for testing
+  // change below params
+  const host = req.protocol + '://' + req.get('host')
+  const url = new URL(host + req.url)
+  const user = url.searchParams.get('user')
+  const group = url.searchParams.get('group')
+  const userId = require('../src/db/user').by_name(user).id
```

Comment:
Is there any good reason for the invite link to be restricted to a specific user? Any user with the link for a group should be able to join that group.

Reply:
Was not clear from the task description, will be changed